src/manifest/representation.ts

```diff
@@ -191,6 +203,15 @@ class Representation {
 
     this.cdnMetadata = args.cdnMetadata;
 
+    if (this.supplementalCodec !== undefined) {
+      const supplementalCodecMimeTypeStr =
+       `${this.mimeType ?? ""};codecs="${this.supplementalCodec ?? ""}"`;
+      const isSupplementalCodecSupported = isCodecSupported(supplementalCodecMimeTypeStr);
```

Comment:
Now that we do this, what would be the point of computing `isCodecSupported` now that `isSupplementalCodecSupported` fills the same role when equal to `true`?

Reply:
Also, I'm not sure we checked other areas of the code using `isSupported` where we now prefer to use `isSupplementalCodecSupported` if now true

Reply:
is  `isCodecSupported()` a costly function in term of performance?

We can eventually skip lines 222 to 231 if `isSupplementalCodecSupported ` is `true` 
and set directly  `this.isSupported` to `true`as the codec **should** be supported because the retro-compatible supplemental codec is supported.

```js
   if (opts.type === "audio" || opts.type === "video") {
      const mimeTypeStr = this.getMimeTypeString();
      const isSupported = isCodecSupported(mimeTypeStr);
      if (!isSupported) {
        log.info("Unsupported Representation", mimeTypeStr, this.id, this.bitrate);
      }
      this.isSupported = isSupported;
    } else {
      this.isSupported = true; // TODO for other types
    }
```

Reply:
> is isCodecSupported() a costly function in term of performance?

Probably not on most devices, yet it's an MSE API we do not control, so I would be more comfortable if we avoid calling it if it's not too costly to do so.

>  and set directly this.isSupported to trueas the codec should be supported because the retro-compatible supplemental codec is supported.

That's true